README.md

```diff
@@ -45,9 +43,9 @@ For more information on using Model Server in various scenarios you can check th
 
 * [Model repository configuration](https://docs.openvino.ai/2022.2/ovms_docs_models_repository.html)
 
-* [Using a docker container](https://docs.openvino.ai/2022.2/ovms_docs_docker_container.html)
+* [Using a docker container](https://docs.openvino.ai/2022.2/ovms_docs_deploying_server.html)
```

Comment:
merge those 2 link to one items - Deployment options

Reply:
Ok.